recipes/capnproto/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: capnproto
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/sandstorm-io/capnproto/archive/v{{ version }}.tar.gz
+  md5: 909bd13ad6b8bc840ac78ab8f5bcb0a4
+
+build:
```

Comment:
Also, the [`boost`]( https://github.com/conda-forge/boost-feedstock/blob/master/recipe/meta.yaml ) recipe is a great place to look for this stuff.